clarity-ext-scripts/clarity_ext_scripts/covid/knm_sminet_service.py

```diff
@@ -0,0 +1,157 @@
+import codecs
+import dateutil.parser
+from datetime import datetime
+from clarity_ext_scripts.covid.partner_api_client import PartnerAPIV7Client
+from sminet_client import SmiNetClient, SmiNetConfig
+from clarity_ext.utils import lazyprop
+from sminet_client import (SampleInfo, ReferringClinic, Patient, Doctor, SmiNetConfig, NotificationType,
+                           LabDiagnosisType, LabResult, Laboratory, SmiNetLabExport)
+from sminet_client import Gender as SmiNetGender
+from clarity_ext_scripts.covid.knm_service import KNMConfig, ServiceRequestProvider
+
+
+class KNMSmiNetIntegrationService(object):
+
+    MAP_GENDER_FROM_KNM_TO_SMINET = {
+        "male": SmiNetGender.MALE,
+        "female": SmiNetGender.FEMALE,
+        "other": SmiNetGender.UNKNOWN,
+        "unknown": SmiNetGender.UNKNOWN,
+        None: SmiNetGender.UNKNOWN,
+    }
+
+    def __init__(self, config):
+        self.config = config
+
+        knm_config = KNMConfig(config)
+        sminet_config = SmiNetConfig(**config)
+
+        self.knm_client = PartnerAPIV7Client(**knm_config)
+        self.sminet_client = SmiNetClient(sminet_config)
+
+    def get_county_from_organization(self, organization):
+        """
+        Given the raw json from the KNM service, returns a county understood by SmiNet
+        """
+        aliases = organization["alias"]
+        for alias in aliases:
+            county_code = alias.split("-")[0]
+            # TODO: THIS IS JUST FOR TEMPORARY TEST PURPOSES (waiting for knm fix)
+            if county_code == "SE":
+                county_code = "AB"
```

Comment:
Should this be removed?

Reply:
Yes, I'll fix it before removing the draft status!